src/entities/mutations/panels-mutations.ts

```diff
@@ -18,7 +18,7 @@ export class PanelsMutations {
     };
 
     const createIdentifierToValue = <T extends object>(value : T) : {value : T} & EditorEntityValue => {
-      console.log(value, '<<<<<<<<<<<<<<<<<');
+      console.log(value, "<<<<<<<<<<<<<<<<<");
```

Comment:
Uhm... Thanks for fixing this eslint issue? But can we remove the log entirely? :rofl:

Reply:
This was changed purely by `eslint --fix`, why I didn't see it. That being said, yes, can be removed